modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -708,12 +713,36 @@ def test_drop():
     df_equals(modin_simple.drop([0, 1, 3], axis=0), simple.loc[[2], :])
     df_equals(modin_simple.drop([0, 3], axis="index"), simple.loc[[1, 2], :])
 
-    pytest.raises(ValueError, modin_simple.drop, 5)
-    pytest.raises(ValueError, modin_simple.drop, "C", 1)
-    pytest.raises(ValueError, modin_simple.drop, [1, 5])
-    pytest.raises(ValueError, modin_simple.drop, ["A", "C"], 1)
+    # TODO(https://github.com/modin-project/modin/issues/5163): raise a
+    # KeyError like pandas when the label is not found when lazy_execution is
+    # off. Also use df_equals instead of
```

Comment:
Complete sentence?

Reply:
I really don't know what I was thinking of. Deleted.